include/iec104.h

```diff
@@ -229,7 +240,7 @@ class IEC104Client
     template <class T>
     void m_addData(std::vector<Datapoint*>& datapoints, int64_t ioa,
                    const std::string& dataname, const T value,
```

Comment:
for consistency rename (or revert the rename in .cpp) value and qd (also in public declaration)